projects/optic/src/commands/capture/init.ts

```diff
@@ -26,50 +26,52 @@ export async function initCaptureConfig(
 // returns a complete Capture block example
 export function captureConfigExample(oasFile: string) {
   return `
+    # Configures the 'optic capture' flow. View the documentation for details about how capture works: https://www.useoptic.com/docs/capturing-traffic
```

Comment:
url doesn't work yet, but will as soon as we publish the corresponding doc updates which should happen shortly.